src/config.rs

```diff
@@ -44,6 +46,30 @@ pub struct ExportersConfig {
     pub influxdb: InfluxDBConfig,
 }
 
+#[derive(Debug, Deserialize, PartialEq)]
+#[serde(rename_all = "lowercase")]
+#[serde(remote = "LevelFilter")]
+pub enum LevelFilterDef {
+    Off,
+    Error,
+    Warn,
+    Info,
+    Debug,
+    Trace,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct LoggerConfig {
+    #[serde(with = "LevelFilterDef")]
+    #[serde(default = "default_log_level_filter")]
```

Comment:
Instead of defining default here, how about if you make this mandatory and define the default value in `uption.toml` example config? Same thing for `enable_stdout`.